src/main/kotlin/org/rust/lang/core/psi/impl/RustFile.kt

```diff
@@ -25,6 +25,8 @@ class RustFile(fileViewProvider: FileViewProvider) : PsiFileBase(fileViewProvide
 val PsiFile.modulePath: RustModulePath?
     get() = RustModulePath.devise(this)
 
+val VirtualFile.isRustFile: Boolean get() = fileType == RustFileType
+val VirtualFile.isNotRustFile: Boolean get() = !isRustFile
```

Comment:
Dubious API decision :) But I think that most uses will be `if (!file.isRustFile) return defualt`, so it makes sense to add a bit of readability here.